packages/jaeger-ui/src/components/TracePage/TraceStatistics/tableValues.tsx

```diff
@@ -209,7 +209,7 @@ function calculateContent(trace: Trace, span: Span, allSpans: Span[], resultValu
 
           tempSelf = onlyOverlay(overlayWithout, allChildrenWithout, tempSelf, span);
           const diff = span.relativeStartTime + span.duration - earliestLongerAsParent.relativeStartTime;
-          tempSelf -= diff;
+          tempSelf = Math.max(0, tempSelf - diff);
```

Comment:
Apparently a bug - the final test case returns a negative value without this guard. The example is somewhat contrived as it hinges on two child spans firing off exactly at trace start time—which is unlikely, but that seems to be the only way to reach line 34.